test/distributed/_composable/fully_shard/test_fully_shard_util.py

```diff
@@ -112,5 +121,32 @@ def test_get_sharded_module_tree_with_module_name_to_fqns(self):
         )
 
 
+class TestUtilsSingleDevice(TestCase):
+    @pytest.mark.skipif(not is_cuda_8_9, reason="requires SM89 compatible machine")
```

Comment:
Same as before, is it sm89+ or only sm89?

Reply:
this is sm89+

Reply:
i can rename to is_sm_89_plus if it helps

Reply:
In that case it might fit in here: https://github.com/pytorch/pytorch/blob/40b09edd87fcbe4e63c4db6399ec758d5c34e1b1/torch/testing/_internal/common_cuda.py#L27

Reply:
got you. will move there